go.mod

```diff
@@ -48,15 +47,15 @@ require (
 	github.com/spf13/cobra v0.0.6
 	github.com/spf13/viper v1.6.2
 	github.com/streadway/quantile v0.0.0-20150917103942-b0c588724d25 // indirect
-	github.com/stretchr/testify v1.4.0
+	github.com/stretchr/testify v1.5.1
```

Comment:
Curios if you need this change here, may worth doing it in a separate PR.

Reply:
Not necessary - will remove 👍